### PR TITLE
Feature/96 partnerships homepage

### PR DESCRIPTION
**Pull request by kevinhowbrook**
Rebased off https://github.com/torchbox/rca-wagtail-2019/pull/116
Needs FE attention for the carousels

### CODE REVIEW

File: rca/home/models.py [hunk 1]
```diff
@@ -1,29 +1,166 @@
...
+        context["hero_colour"] = "dark"
+        if self.hero_colour_option == "1":
+            context["hero_colour"] = "light"
+
+        context["transformation_blocks"] = self.transformation_blocks.select_related(
+            "image"
+        )
+        context["partnerships_block"] = HomePagePartnershipBlock.objects.filter(
+            source_page=self
+        ).first
```
Comment (tm-kn) [inline rca/home/models.py]: Does this even work? Shouldn't it be `self.partnerships_block.first()` or `self.partnerships_block.all().first()`?

File: rca/project_styleguide/templates/patterns/pages/home/home_page.html [hunk 2]
```diff
@@ -13,29 +13,35 @@
...
-            <section class="section section--top-space bg bg--light">
-                {% include "patterns/organisms/image-video-block/image-video-block.html" with modal=False heading=image_video_block.heading subheading=image_video_block.subheading has_meta=True %}
-
-                <div class="section__notch">
-                    <div class="section__notch-fill section__notch-fill--third-col"></div>
-                </div>
-            </section>
+                </section>
+            {% endif %}
+            {% if transformation_blocks.first %}
```
Comment (tm-kn) [inline rca/project_styleguide/templates/patterns/pages/home/home_page.html]: I am bamboozled why can't you just pass one instance from the view if you will only display one.

**Comment by kevinhowbrook** [comment]
Thanks @tm-kn, I'll address these points on https://github.com/torchbox/rca-wagtail-2019/pull/119

File: rca/home/models.py (hunk 1 above)
Comment (kevinhowbrook) [reply rca/home/models.py]: This does work, but again I don't need the full queryset here, your recommendation of `self.partnerships_block.first()` works fine and is more simple, thanks!

File: rca/project_styleguide/templates/patterns/pages/home/home_page.html (hunk 2 above)
Comment (kevinhowbrook) [reply rca/project_styleguide/templates/patterns/pages/home/home_page.html]: Thanks Tomasz, changed on https://github.com/torchbox/rca-wagtail-2019/pull/119
```
context["transformation_block"] = self.transformation_blocks.select_related(
            "image"
        ).first()```

**Pull request closed without merging**